data/json/items/tools.json

```diff
@@ -3223,6 +3223,7 @@
     "description": "This is a glass vial, split into two chambers.  The divider is removable, which will cause the chemicals to mix.  If this mixture is exposed to air (perhaps by throwing the vial) they will spill out as a pool of potent acid.",
     "price": 5000,
     "material": "glass",
+    "flags": "PROCESS_ON_DROP",
```

Comment:
I wanted it to be something more generic, so that it could be used both for ammo and thrown items. Also, "ON_THROW" could suggest that it is processed in mid-air.
"ON_HIT" could work.